docs/examples/irradiance-transposition/use_perez_modelchain.py

```diff
@@ -0,0 +1,123 @@
+"""
+Use different Perez coefficients with the ModelChain
+====================================================
+
+This example demonstrates how to customize the ModelChain
+to use site-specifc Perez coefficients.
+"""
+
+# %%
+# The :py:class:`pvlib.modelchain.ModelChain` object provides a useful method
+# for easily constructing a model with a simple, unified interface.
+# However, a user may want to customize their models in various ways.
+# One such example is during the irradiance transposition step.
+# The Perez models perform very well on field data, but
```

Comment:
Is there a reason this is plural "Perez models"? I think the example is relevant to only one Perez model.